app/cdap/components/Snackbar/Components/Transition/index.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import CheckCircleOutlinedIcon from '@material-ui/icons/CheckCircleOutline';
+import { Box, Typography } from '@material-ui/core';
+import { useStyles } from './styles';
+import CloseIcon from '@material-ui/icons/Close';
+import T from 'i18n-react';
+import ErrorOutlineIcon from '@material-ui/icons/ErrorOutline';
+import { ITransitionComponentProps } from './types';
```

Comment:
as this is a new file, add relative import for all here

Reply:
not resolved, please check this comment

Reply:
resolved